use-cases.md

```diff
@@ -0,0 +1,50 @@
+## Use-Cases for WebTransport
+
+Ordinality of listing does not imply priority. 
+
+1. **Machine learning**
+    1. Speech translation/emotion analysis  - sending audio/video data from client to server for analysis and receiving translated data/text/audio in return.
+    1. Security camera analysis - data and/or video sent to cloud service for analysis. Service may return data instructions.
```

Comment:
This use case may need clarification. Is this a p2p situation, a non-web-based "security camera" talking to a server, or a local camera connected to a browser client? Unless it's the latter it may be out of scope.

Reply:
ok if web-based device.